### ISSUE TITLE
Make an "awards" page

### ISSUE THREAD

**Issue by andrewda** [SouthEugeneRoboticsTeam/SouthEugeneRoboticsTeam.github.io#56]
It might be cool to have a list of awards (and blue banners) our team has won.

**Comment by andrewda** [comment]
It'd be really cool if someone wrote a GitHub Action to automatically update the list of awards from The Blue Alliance!

**Comment by will-hou** [comment]
Good idea! I think the "Robots" page might be an okay place to put them. Not quite sure how well it'll end up looking but we can always just list them in text form if it doesn't work out

Potential layout:

![sample_awards_page](https://user-images.githubusercontent.com/23201789/66800681-fc95f000-eeca-11e9-8c55-95c9c2c0d94f.png)

**Comment by andrewda** [comment]
Ooooh I really like that idea
